src/internal-api.yml

```
@@ -210,6 +210,37 @@ paths:
               application/json:
                 schema:
                   $ref: data/result/find.yml
+    "/app-api/v1/data/colls/{coll}/insertMany":
+      post:
+        tags:
+          - Data
+        summary: Inserts many documents in database
+        operationId: insertManyDocuments
+        parameters:
+          - name: coll
+            in: path
+            description: The documents collection name
+            required: true
+            schema:
+              type: string
+        requestBody:
+          description: The documents to create
+          required: true
+          content:
+            application/json:
+              schema:
+                type: array
+                items: 
+                  type: object
```

The documents should by in the `documents` field of an object to be able to handle the other parameters like it's done for updateMany: https://www.mongodb.com/docs/manual/reference/method/db.collection.insertMany/

```suggestion
                type: object
                properties:
                  documents:
                    type: array
                    items: 
                      type: object
                      additionalProperties: true
                required:
                  - documents
                additionalProperties: false
```